docs/notebook/import_data.rst

```diff
@@ -48,3 +48,21 @@ For Imviz::
     imviz = Imviz()
     imviz.app
     imviz.load_data("/path/to/data/image.fits")
+
+
+Importing Custom Line Lists
+===========================
+
+Jdaviz comes with curated line lists built by the scientific community. If you cannot find the lines you need, you can add your own by constructing an :ref:`astropy table <astropy:construct_table>`! For example::
```

Comment:
Nitpick: It is easier for future review if you break this into mutiple lines. Also, I don't think the exclamation point is necessary.

```suggestion
Jdaviz comes with curated line lists built by the scientific community.
If you cannot find the lines you need, you can add your own by constructing
an :ref:`astropy table <astropy:construct_table>`; For example::
```